casepro/backend/junebug.py

```diff
@@ -115,7 +188,28 @@ def pull_contacts(
             tuple of the number of contacts created, updated, deleted and
             ignored
         """
-        return (0, 0, 0, 0)
+        identity_store = self.identity_store
+
+        # all identities created in the Identity Store in the time window
+        new_identities = identity_store.get_identities(
+            created_at__gte=modified_after, created_at__lte=modified_before)
```

Comment:
The issue I see here is that I don't think that the identity store supports multiple arguments on its `search` endpoint. That should probably be fixed in the identity store, so we should probably create an issue there (also, this sync probably won't work properly since currently the identity store will only filter on the first argument).